lineage/CellVar.py

```diff
@@ -165,7 +165,7 @@ def double(parent_state, T):
 
     # The index of the 1 value in the resulting list in the two trials represent the state that was picked for that
     # dice toss.
-    return left_state_results[1], right_state_results[1]
```

Comment:
Though this is faster than the proposed method, this method only matches the results we want when our lineage tree only has two states. When we have more than three states, this won't work, and a lot of cells will be made 0 inccorrectly.

Reply:
![image](https://user-images.githubusercontent.com/20716101/83312954-26392880-a1c9-11ea-899e-f14b5f323e57.png)

For example, the existing method would not be able to say that the cells should be state 2 for some of these possible transition matrix rows with 3 states.